src/herder/TransactionQueue.cpp

```diff
@@ -736,8 +736,8 @@ TransactionQueue::broadcastTx(AccountState& state, TimestampedTx& tx)
 #endif
     tx.mBroadcasted = true;
     state.mBroadcastQueueOps -= tx.mTx->getNumOperations();
-    return mApp.getOverlayManager().broadcastMessage(
-        tx.mTx->toStellarMessage());
+    return mApp.getOverlayManager().broadcastMessage(tx.mTx->toStellarMessage(),
+                                                     nullptr);
```

Comment:
Based on a discussion with @MonsieurNicolas, we decided to finish tracking early in this first version, because the transaction queue itself already has throttling/load shedding mechanisms. Also, since the transaction queue implements some prioritization policy as well, we might get very noisy signals if we track the transaction all the way to broadcasting phase. On the other hand, current definition of "amount of work per transaction" might be too optimistic, since we do not include the write time at all. So I think we can have a subsequent change after this PR that improves the accounting of work for transactions.

Reply:
Ok.